src/common/nativelibrary_for_netstandard2.cs

```diff
@@ -370,6 +370,12 @@ LibSuffix suffix
                 a.Add(Path.Combine(dir, libname));
             }
 
+            if ((flags & WHERE_CODEBASE) != 0) 
+            {
+               var dir = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().CodeBase);
```

Comment:
[Assembly.CodeBase](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.codebase) is obsolete and throws on .NET 5 and later version.

Reply:
Vsto apps are preserved in amber on the 4.0 runtime, but i get your point. I could limit this to .net framework with `System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription‘